autokeras/hypermodel/hyper_block.py

```diff
@@ -101,9 +101,111 @@ def build(self, hp, inputs=None):
 
 
 class XceptionBlock(HyperBlock):
+    """ XceptionBlock
+    An Xception structure, used for specifying your model with specific datasets.
+
+    The original Xception architecture is from https://arxiv.org/abs/1610.02357.
+    The data first goes through the entry flow, then through the middle flow which
+    is repeated eight times, and finally through the exit flow.
+
+    This XceptionBlock returns a similar architecture as Xception except without
+    the last (optional) fully connected layer(s) and logistic regression.
+    The size of this architecture could be decided by `HyperParameters`, to get an
+    architecture with a half, an identical, or a double size of the original one.
+    """
 
     def build(self, hp, inputs=None):
-        pass
+        input_node = layer_utils.format_inputs(inputs, self.name, num=1)[0]
+        output_node = input_node
+        channel_axis = 1 if tf.keras.backend.image_data_format() == 'channels_first' else -1
+
+        # To decide the size of the architecture
+        filter_entry = hp.Choice('num_filters', [16, 32, 64], default=32)
+        filter_middle = int(filter_entry * 22.75)
+        filter_exit = int(filter_entry * 48)
+        # To decide the pooling layer in the end
+        last_pooling = hp.Choice('pooling', ['avg', 'max'], default='avg')
```

Comment:
We don't need the pooling in the block.
So just delete the pooling layers and this variable.